crates/bevy_render/src/render_graph/nodes/render_resources_node.rs

```diff
@@ -482,23 +479,6 @@ fn render_resources_node_system<T: RenderResources>(
         state
             .uniform_buffer_arrays
             .copy_staging_buffer_to_final_buffers(&mut state.command_queue, staging_buffer);
-    } else {
-        // TODO: can we just remove this?
```

Comment:
It seems it can be safely removed, tested a lot of bevy examples without it and everything seems to work.